scene/gui/color_picker.h

```diff
@@ -73,12 +90,22 @@ class ColorPicker : public BoxContainer {
 		SHAPE_MAX
 	};
 
+	static const int SLIDER_COUNT = 4;
+
 private:
 	static Ref<Shader> wheel_shader;
 	static Ref<Shader> circle_shader;
 	static Ref<Shader> circle_ok_color_shader;
 	static List<Color> preset_cache;
 
+	int current_slider_count = SLIDER_COUNT;
+
+	bool slider_theme_modified = true;
+	PickerShapeType last_shape = SHAPE_MAX;
+
+	ColorMode *selected_mode;
```

Comment:
```suggestion
	ColorMode *selected_mode = nullptr;
```
All pointers should be null-initialized.